tests/integration/test_tiler.py

```diff
@@ -117,55 +150,88 @@ def test_extract_tiles_respecting_the_given_tile_size(
 
 class DescribeGridTiler:
     @pytest.mark.parametrize(
-        "fixture_slide, level,check_tissue, expectation",
+        "fixture_slide, binary_mask, tile_size, level,check_tissue, expectation",
```

Comment:
why here all the tile sizes are squared?

Reply:
I can change that, but is it important?

Reply:
it is probably important as it is with the random tiler

Reply:
kkkk